probable-futures.md

```diff
@@ -15,6 +15,12 @@ This document includes three tilesets
 
 Probable Futures is an initiative that aims to increase the chances that our future is good. We offer useful tools to visualize climate change along with stories and insights to help people understand what those changes mean. We also partner with courageous leaders and influential organizations — without cost or compensation — to show how this perspective and these tools can be applied towards practical approaches to the problems posed by climate change. 
 
+## Attribution 
+
+When publishing any Probable Futures datasets, please include the following credit:
```

Comment:
@petercroce would it be more appropriate to say "When publishing a map using any Probable Futures dataset, please include the following credit:"?

Reply:
Yes! Good change